<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletCacheDebugDialog</class>
 <widget class="QDialog" name="WalletCacheDebugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1423</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wallet Cache Debug</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QPlainTextEdit" name="output">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="m_blockchain">
         <property name="text">
          <string>m_blockchain</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_transfers">
         <property name="text">
          <string>m_transfers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_key_images">
         <property name="text">
          <string>m_key_images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_pub_keys">
         <property name="text">
          <string>m_pub_keys</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_address_book">
         <property name="text">
          <string>m_address_book</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="m_unconfirmed_payments">
         <property name="text">
          <string>m_unconfirmed_payments</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_payments">
         <property name="text">
          <string>m_payments</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_unconfirmed_txs">
         <property name="text">
          <string>m_unconfirmed_txs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_confirmed_txs">
         <property name="text">
          <string>m_confirmed_txs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_tx_keys">
         <property name="text">
          <string>m_tx_keys</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_additional_tx_keys">
         <property name="text">
          <string>m_additional_tx_keys</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QRadioButton" name="m_tx_notes">
         <property name="text">
          <string>m_tx_notes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_scanned_pool_txs">
         <property name="text">
          <string>m_scanned_pool_txs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_subaddresses">
         <property name="text">
          <string>m_subaddresses</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_subaddress_labels">
         <property name="text">
          <string>m_subaddress_labels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_account_tags">
         <property name="text">
          <string>m_account_tags</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_attributes">
         <property name="text">
          <string>m_attributes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WalletCacheDebugDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WalletCacheDebugDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
